src/main/xar-resources/data/backup/backup.xml

```diff
@@ -442,6 +442,11 @@
             <programlisting xlink:href="listings/listing-9.txt"/>
             <para>Default settings for the user, password or server URIs can also be set in the
                 <literal>backup.properties</literal> file.</para>
+            <para>If you want to connect to a databse using SSL, use the parameter <literal>--use-ssl</literal> or
```

Comment:
Maybe we should avoid sentences with "you" ...

Reply:
`backup.xml` contains lots of phrases with "you", so I oriented myself on that. We could change that to "To connect to a databse using SSL"...?

Reply:
@zed-g I think its okay as you have followed the existing style. We should likely do a bigger job to improve the overall style